interfaces/apparmor/spec.go

```diff
@@ -136,13 +136,30 @@ func (spec *Specification) AddDeduplicatedSnippet(snippet string) {
 	}
 }
 
-func (spec *Specification) AddParametricSnippet(template, value string) {
-	if !strings.Contains(template, "###PARAM###") {
-		panic(fmt.Sprintf("template %q does not contain ###PARAM###", template))
-	}
+func (spec *Specification) AddParametricSnippet(templateFragment []string, value string) {
 	if len(spec.securityTags) == 0 {
 		return
 	}
+
+	// We need to build a temlate string from the templateFragment.
```

Comment:
this seems to belong to a doc comment (needed), not a comment only inside the function

Reply:
Indeed, plus there is a typo in my comment, it should be "tem*p*late"

Reply:
Done